spec/abstractops.html

```diff
@@ -1796,7 +1796,7 @@ <h1>
     <h1>
       GetDifferenceSettings (
         _operation_: ~since~ or ~until~,
-        _options_: an ECMAScript language value,
+        _options_: an Object,
```

Comment:
should there be some kind of assertion here that this is an ordinary object with only data properties whose [[Prototype]] is null?

Reply:
I think it's fine without that. The operation remains meaningful regardless, we're just choosing not to employ it in that way.